website/src/categories/components/component-table.njk

```diff
@@ -7,7 +7,7 @@
             <section>
             <h2 id={{ tagName.selector }} tabindex="-1">
               <a class="header-anchor" href="#{{ tagName.selector }}">
-                {%- if component.data.componentTagName.length > 1 -%}{{ tagName.name }}{% endif %} API
+                {%- if component.data.componentTagName.length > 1 -%}{{ tagName.name }}{%- endif -%} API
```

Comment:
Space is missing between component name and API? :)

<img width="238" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/7e71e0bf-e5a7-4e25-9c26-a0cbed2e42f6">



and 


<img width="199" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/0aac1397-b0c2-42be-85c8-00792e4d397c">